Sources/LocationMessageDisplayDelegate.swift

```diff
@@ -50,6 +50,19 @@ public protocol LocationMessageDisplayDelegate: MessagesDisplayDelegate {
     /// - Returns: Your customized MKAnnotationView or nil to not show any.
     func annotationViewForLocation(message: MessageType, at indexPath: IndexPath, in messageCollectionView: MessagesCollectionView) -> MKAnnotationView?
 
+
+    /// Ask the delegate for a custom animation block to run when whe map screenshot is ready to be displaied in the given location message
+    /// The animation block is called with the view to be animated. You can animate it with CoreAnimation, UIView.animate or any library you prefer.
+    ///
+    /// default: nil
+    ///
+    /// - Parameters:
+    ///   - message: The location message with the map to animate
+    ///   - indexPath: Message's index path
+    ///   - messagesCollectionView: The collection view requesting the information
+    /// - Returns: Your customized animation block.
+    func animationBlockForLocation(message: MessageType, at indexPath: IndexPath, in messagesCollectionView: MessagesCollectionView) -> ((UIView) -> Void)?
```

Comment:
I think that the `LocationMessageCell`s `ContentView` is a `UIImageView` and not a `UIView`? We may want to represent this in the block

Reply:
Sure! I will change that 🙂